docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -10,6 +10,10 @@ building, and decide what image to build. Currently there are two versions:
 being a version that includes `Qt Automotive Suite`_ components that enable the
 NeptuneUI demo application.
 
+Both images are available as `-dev` versions, which, for example, sets an empty
+root password and allows empty password login via ssh. This is usually useful
+during development, unless one explicitly sets a root password elsewhere in the
+build.
```

Comment:
Maybe a stupid question, but how does one set a root password elsewhere in the build?

Reply:
Hmmm I just realized this isn't exactly true anymore and I should probably update the text anyway. Will add notes about how to set root pw.